modules/gitops-dex-role-mappings.adoc

```diff
@@ -0,0 +1,21 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="gitops-dex-role-mappings_{context}"]
+= Mapping users to specific roles
+
+Argo CD cannot map to users if they have a direct `ClusterRoleBinding` role. You can manually change users' role as `role:admin` on SSO through OpenShift.
+
+.Procedure
+
+. Create a group named `cluster-admins`.
+. Add the user to the group.
+. Apply the `cluster-admin` `ClusterRole` to the group:
++
+[source,terminal]
+----
+  $ oc adm groups new cluster-admins
+  $ oc adm groups add-users cluster-admins USER
+  $ oc adm policy add-cluster-role-to-group cluster-admin cluster-admins
```

Comment:
Please note the [OCP guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output): Do not use more than one command per code block. For example, the following must be split up into three separate code blocks: See the example provided.
However in this case it is clear that each command should be under the respective steps. Please add the three commands under each step.